gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/proc/KillDagProc.java

```diff
@@ -48,14 +47,11 @@
  */
 @Slf4j
 public class KillDagProc extends DagProc<Optional<Dag<JobExecutionPlan>>> {
-  private final Optional<DagNodeId> dagNodeId;
+  private final boolean dagNodeKillRequest;
```

Comment:
we've inherited potential confusion because "job" and "dagNode" are basically synonyms.  perhaps we should use "job" for what's exposed to users (e.g. API endpoints and the DagActions" directly created from those) and "dagNode" for internal mechanics of carrying out those actions.  in this case, I'd opt for "job" terminology, so maybe `shouldKillSpecificJob`

Reply:
but this flag is for internal purpose, right?